Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -251,6 +257,14 @@ public class Snapshotter {
                                 bearing: bearing?.NSNumber,
                                 pitch: pitch?.NSNumber))
     }
+
+    // MARK: - Telemetry
+    internal func sendTurnstileEvent() {
+        let accessToken = ResourceOptionsManager.default.defaultAccessToken()
```

Comment:
Snapshotter is initialized with `MapSnapshotOptions` which have `ResourceOptions`. We should use the access token from there instead.